__tests__/app.test.js

```diff
@@ -8,34 +8,95 @@ beforeEach(() => seed(testData));
 
 afterAll(() => db.end());
 
-describe("/api/categories", () => {
+describe("GET /api/categories", () => {
   test("200: returns array", () => {
     return request(app)
       .get("/api/categories")
       .expect(200)
-      .then(({ body }) => {
-        expect(body).toBeInstanceOf(Array);
+      .then(({ body: { categories } }) => {
+        expect(categories).toBeInstanceOf(Array);
       });
   });
   test("200: array contains objects with slug and decription keys", () => {
     return request(app)
       .get("/api/categories")
       .expect(200)
-      .then(({ body }) => {
-        body.forEach((category) => {
+      .then(({ body: { categories } }) => {
+        categories.forEach((category) => {
```

Comment:
whenever you are using a `forEach` in your tests have a think about what would happen if the array was empty...